### PR TITLE
fix: `is` not working with upper/mixed case values

### LINKED ISSUE #2077: `is` operator no longer working with an uppercase value
Unintended effect of https://github.com/PostgREST/postgrest/pull/2027:
```http
GET /projects?id=is.NULL
```

```js
{"details":"unexpected \"N\" expecting null or trilean value (unknown, true, false)","message":"\"failed to parse filter (is.NULL)\" (li
ne 1, column 4)"}
```

### PR DESCRIPTION
**Pull request by steve-chavez**
Fixes #2077.

### CODE REVIEW

File: test/Feature/QuerySpec.hs [hunk 1]
```diff
@@ -85,6 +85,19 @@ spec actualPgVersion = do
         [json| [{"id": 3, "name": "wash the dishes", "done": null }] |]
         { matchHeaders = [matchContentTypeJson] }
 
+    it "matches with trilean values in upper or mixed case" $ do
```
Comment (laurenceisla) [inline test/Feature/QuerySpec.hs:88]: Maybe it needs a test for `NULL` (not here, but in the "matches nulls..." section) for completion.

**Review by laurenceisla — APPROVED** [review]
